eslint.config.js

```diff
@@ -1,7 +1,8 @@
 const eslint = require("@eslint/js");
 const n = require("eslint-plugin-n");
+const tsESLint = require("typescript-eslint");
 
-module.exports = [
+module.exports = tsESLint.config([
```

Comment:
[Style] `tseslint.config` flattens arrays for us ✨ 

```suggestion
module.exports = tsESLint.config(
```

Reply:
updated